proposal.rst

```diff
@@ -239,7 +301,82 @@ Project Timeline
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-FILLME
+MONTH 1  : 
+
+BIG TASK NAME : DOCUMENTATION STEP
+
+Week 1 
+	- Read and understand code-base further. 
+	- Create a new VagrantFile and make sure that it installs all the xapian dependencies. I have a doc where I have all commands which helped me with my Xapian setup.
```

Comment:
Why do we need to create a new `VagrantFile`?

If ceb80bbe06af6156f30e2553739b6f94a892f712 didn't fully fix it, it's likely to be less work to iteratively fix what we have rather than throw everything out and start again.